### PR TITLE
cluster-capi-operator: setup ci jobs

### PR DESCRIPTION
**Pull request by alexander-demicev**

### CODE REVIEW

**Comment by alexander-demichev** [comment]
/assign @smarterclayton

@smarterclayton I added a new component that we are planning to release in 4.10 https://github.com/openshift/meta-cluster-api-operator

File: core-services/image-mirroring/openshift/mapping_origin_4_10 [hunk 1]
```diff
@@ -159,6 +159,8 @@ registry.ci.openshift.org/origin/4.10:loki quay.io/openshift/origin-loki:4.10 qu
 registry.ci.openshift.org/origin/4.10:machine-api-operator quay.io/openshift/origin-machine-api-operator:4.10 quay.io/openshift/origin-machine-api-operator:4.10.0 quay.io/openshift/origin-machine-api-operator:latest
 registry.ci.openshift.org/origin/4.10:manila-csi-driver-operator-test quay.io/openshift/origin-manila-csi-driver-operator-test:4.10 quay.io/openshift/origin-manila-csi-driver-operator-test:4.10.0 quay.io/openshift/origin-manila-csi-driver-operator-test:latest
 registry.ci.openshift.org/origin/4.10:mdns-publisher quay.io/openshift/origin-mdns-publisher:4.10 quay.io/openshift/origin-mdns-publisher:4.10.0 quay.io/openshift/origin-mdns-publisher:latest
+registry.ci.openshift.org/origin/4.10:meta-cluster-api-operator quay.io/openshift/origin-meta-cluster-api-operator:4.10 quay.io/openshift/origin-meta-cluster-api-operator:4.10.0 quay.io/openshift/origin-meta-cluster-api-operator:latest
```
Comment (smarterclayton) [inline core-services/image-mirroring/openshift/mapping_origin_4_10:162]: Naming review:

Ok, why is this called "meta"?  From a naming perpective I don't understand what "meta" means, so if you can articulate that it'll help me review.

**Review by JoelSpeed — COMMENTED** [review]
/lgtm

File: core-services/image-mirroring/openshift/mapping_origin_4_10 [hunk 2]
```diff
@@ -43,6 +43,8 @@ registry.ci.openshift.org/origin/4.10:cluster-autoscaler-operator quay.io/opensh
 registry.ci.openshift.org/origin/4.10:cluster-baremetal-operator quay.io/openshift/origin-cluster-baremetal-operator:4.10 quay.io/openshift/origin-cluster-baremetal-operator:4.10.0 quay.io/openshift/origin-cluster-baremetal-operator:latest
 registry.ci.openshift.org/origin/4.10:cluster-bootstrap quay.io/openshift/origin-cluster-bootstrap:4.10 quay.io/openshift/origin-cluster-bootstrap:4.10.0 quay.io/openshift/origin-cluster-bootstrap:latest
 registry.ci.openshift.org/origin/4.10:cluster-capacity quay.io/openshift/origin-cluster-capacity:4.10 quay.io/openshift/origin-cluster-capacity:4.10.0 quay.io/openshift/origin-cluster-capacity:latest
+registry.ci.openshift.org/origin/4.10:cluster-capi-operator quay.io/openshift/origin-cluster-capi-operator:4.10 quay.io/openshift/origin-cluster-capi-operator:4.10.0 quay.io/openshift/origin-cluster-capi-operator:latest
+registry.ci.openshift.org/origin/4.10:cluster-capi-operator-src quay.io/openshift/origin-cluster-capi-operator-src:4.10 quay.io/openshift/origin-cluster-capi-operator-src:4.10.0 quay.io/openshift/origin-cluster-capi-operator-src:latest
```
Comment (smarterclayton) [inline core-services/image-mirroring/openshift/mapping_origin_4_10:47]: i missed this during review, why are we publishing a source image like this?  what does it enable?

Comment (alexander-demichev) [reply core-services/image-mirroring/openshift/mapping_origin_4_10:47]: There is an enhancement that describes what it does https://github.com/openshift/enhancements/pull/913/files#diff-a0672895c853aefb4deaafb651c091b4e24d5dc2648e340e8e85deb4de58e564R100

**Pull request merged**